src/assets/i18n/en.json

```diff
@@ -17,6 +17,22 @@
   "SAVE": "Save",
   "NO": "No",
   "YES": "Yes",
+  "MESSAGES": "Messages",
+  "APPOINTMENTS": "Appointments",
+  "PROJECTS": "Projects",
+  "TASKS": "Tasks",
+  "RECENT_ACTIVITY": "Recent Activiy",
+  "PROFILE": "My profile page",
```

Comment:
My Profile Page - auf Englisch wird fast jedes Wort in einem Titel großgeschrieben

Reply:
oh nice.. gefällt mir auch besser (bei musik titeln mache ich das auch so, weil ich das ästhetischer finde, und da wurde mir immer gesagt es sei falsch fast alles groß zu schreiben hahaha)